Dockerfile

```diff
@@ -5,5 +5,5 @@ RUN mkdir /app
 WORKDIR /app
 ADD Gemfile /app/Gemfile
 ADD Gemfile.lock /app/Gemfile.lock
-RUN bundle install -j4
+RUN bundle install --path vendor/bundle -j4
```

Comment:
I want to keep dependent libraries in `volumes`. By default installed on system ruby.
So, we add `- /app/vendor` to` volumes` of `docker-compose.yml`.
Of course, this is still cached.

The greatest effect of this is to `bundle install` if a gem is added, but you do not have to recreate the docker image.
Because dependent libraries are persisted in `volumes`.